packages/slice-machine/src/hooks/useEditorContent.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentUI } from "@lib/models/common/ComponentUI";
+import { defaultSharedSliceContent } from "@prismicio/editor-fields";
+import { renderSliceMock } from "@prismicio/mocks";
+import { SharedSliceContent } from "@prismicio/types-internal/lib/documents/widgets/slices";
+import { Slices } from "@slicemachine/core/build/models";
+
+function useEditorContentOnce({
+  variationID,
+  slice,
+}: {
+  variationID: string;
+  slice: ComponentUI;
+}) {
+  const editorContent =
+    slice.mock?.[0] ||
+    (defaultSharedSliceContent(variationID) as SharedSliceContent);
```

Comment:
`defaultSharedSliceContent` has nothing to do in `editor-fields` actually, it only returns a `SharedSliceContent which you can access here so you should simply create the function here or return directly the right json, but it's a useless dependency to create here on the content level.
The goal of the editor is to be stateless, get some json as input and output an updated json. So it should never provide any form of default data or data whatsoever.